### DISCUSSION TITLE
Date format for variables?

### DISCUSSION THREAD

**Discussion by zypA13510** [PSAppDeployToolkit/PSAppDeployToolkit#979] [Q&A] [answered]
https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/blob/dc3c003bbb0e78f025e98b96c3fc9c6e672e9c3e/Toolkit/Deploy-Application.ps1#L116C5-L116C42
```powershell
[String]$appScriptDate = 'XX/XX/20XX'
```

Sorry but it is not clear if the date should be given in MM/DD/YYYY or DD/MM/YYYY, or does it not matter at all (tbh I'd prefer YYYY-MM-DD as it is the ISO format)? What is the purpose of this variable, is it informational only?

**Comment by DanGough** [comment, marked as answer]
You can use whatever format you like; it's only stored as a string rather than a datetime object, and only referenced in one place in the code to just print the value to the log file:

```ps
If ($appScriptDate) {
    Write-Log -Message "[$installName] script date is [$appScriptDate]" -Source $appDeployToolkitName
}
```
